api_docs/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 9.0
 
+**Feature level 255**
+* [`GET /events`](/api/get-events),
```

Comment:
You're missing a blank line here, which makes it misrender.

It's a bit confusing, because Zulip's own Markdown doesn't require that blank line, but most do, including the ones we use for documentation.

Reply:
I have fixed that now.